### DISCUSSION TITLE
혹시 샴 tts emoji끄기 방법이 있나요?

### DISCUSSION THREAD

**Discussion by CHIHIT** [WoodNeck/siamese#6] [Q&A]
tts 사용할때마다 음표 emoji가 붙어서 채팅이 너무 길어져서 끄는방법이 있나 궁금해서 질문해봅니다

**Comment by WoodNeck** [comment]
@CHIHIT 
![image](https://user-images.githubusercontent.com/26213435/132269760-9ee15d50-4b0d-489e-b624-beb78d50fa26.png)
위 이모지를 말씀하시는거라면 따로 방법을 설정해두진 않았습니다.
필요하시면 만들도록 할게요~
